src/ex1/script.js

```diff
@@ -0,0 +1,158 @@
+const todoListElement = document.getElementById("todo-list")
+const addTodoButton = document.getElementById("add-todo-button")
+const todoInput = document.getElementById("todo-input")
+const clearAllTodosButton = document.getElementById("clear-all-todos-button")
+const sumTodos = document.getElementById("sum-todos")
+const orderSelect = document.getElementById("order-select")
+const enterTodos = document.getElementById("enter-todos")
+
+let todoList = [] //array for storing data
+
+showTodos()
+
+todoInput.onkeyup = (e) => {
+    let enterValue = todoInput.value
+    if (enterValue.trim() !== ""){
+        addTodoButton.classList.add("active")
+        addTodoButton.style.cursor = "pointer"
+        addTodoButton.style.opacity = 1
+
+        if(e.keyCode === 13){
+            addTodo()
+        }
+    }
+    else{
+        addTodoButton.classList.remove("active")
```

Comment:
it's considered a good practice to hide the implementation of different actions to improve code readability, for example:

const isEmptyTodoValue = e.keyCode === ENTER_KEY;
if (isEmptyTodoValue) {
    deactivateAddTaskBtn();
} else {
    activateAddTaskBtn();
   if (e.keyCode === ENTER_KEY) {
       addTodo();
   }
}